### PR TITLE
Do not merge `pod_target_xcconfig` from test specs into non test xcconfigs

### PR DESCRIPTION
**Pull request by dnkoutso**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -127,12 +127,18 @@ def fixture_target_definition(name = 'Pods', platform = Pod::Platform.ios)
 
 def fixture_pod_target(spec_or_name, target_definitions = [])
   spec = spec_or_name.is_a?(Pod::Specification) ? spec_or_name : fixture_spec(spec_or_name)
+  fixture_pod_target_with_specs([spec], target_definitions)
+end
+
+def fixture_pod_target_with_specs(specs, target_definitions = [])
```
Comment (dnkoutso) [inline spec/spec_helper.rb]: helper method that allows creating fixture pod targets with multiple specs :D

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please rebase to get rid of the merge commits in this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by dnkoutso** [comment]
rebased

**Comment by orta** [comment]
👍

**Pull request merged**